docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -32,14 +34,12 @@
 
 ## Build
 
-- From a terminal at your application's root directory, `npm run build`. This will create a deployment-ready build in the "build" folder within the application's root directory. It is not necessary to build the application during development.
+- From a terminal at your application's root directory, `npm run build`. This will create a deployment-ready build in the "dist" folder within the application's root directory. You may run `npm run preview` to start a local server to preview your production build. It is not necessary to build the application during development.
 
 ## Useful Links
 
 - [iTwin Viewer React](https://www.npmjs.com/package/@itwin/web-viewer-react)
-- [iTwin Viewer Create React App Template](https://www.npmjs.com/package/@itwin/cra-template-web-viewer)
-- [Bentley React Scripts](https://www.npmjs.com/package/@bentley/react-scripts)
-- [Create React App](https://create-react-app.dev/)
+- [Vite](https://vite.dev/guide/)
```

Comment:
I think we can still add a link to the template thats on [GitHub](https://github.com/iTwin/viewer/tree/master/packages/templates/web), that'd be useful if users want to create issues about the template or contribute. Same with desktop viewer